test/jdk/sun/security/tools/keytool/ReadJar.java

```diff
@@ -159,7 +159,8 @@ private static void testCertOutput() throws Throwable {
                 .shouldContain("Certificate #1:")
                 .shouldContain("Certificate #2:")
                 .shouldContain("Signer #2:")
-                .shouldMatch("The certificate #.* of signer #.*" + "uses the SHA1withRSA.*will be disabled")
+                .shouldNotMatch("The certificate #.* of signer #.*" + "uses the SHA1withRSA.*will be disabled")
```

Comment:
You probably don't need to check for a non-occurrence here since the message has been changed and can no longer occur.

Reply:
Removed the checking for a non-occurrence from the tests.